js/resizer.js

```diff
@@ -54,15 +47,35 @@ async function resizeImage(imgBlob, resizeSettings = {}) {
     width *= maxHeight / height;
     height = maxHeight;
   } else {
-    return new Promise(resolve => resolve(imgBlob));
+    return null;
```

Comment:
if the logic falls into this `else` section, does that meant that there are no resizing done?
if there are no resizing done, then we should also return the image right?

Reply:
If this method is exposed, then technically that should be the behaviour.
However this method is only called internally and returning null here allows to skip an extra step of converting the dataURI back to blob for the SDK in case no resizing happens.

Reply:
ic ok